back/engines/free/frontend/app/services/frontend/url_service.rb

```diff
@@ -131,13 +131,7 @@ def admin_ideas_url(configuration = app_config_instance)
     end
 
     def admin_project_url(project_id, configuration = app_config_instance)
-      project = Project.find(project_id)
-      last_phase_id = project ? TimelineService.new.current_or_backup_transitive_phase(project)&.id : nil
-      if last_phase_id
-        "#{configuration.base_frontend_uri}/admin/projects/#{project_id}/phases/#{last_phase_id}/ideas"
-      else
-        "#{configuration.base_frontend_uri}/admin/projects/#{project_id}/settings"
-      end
```

Comment:
Going to settings was confusing for users (ie there is no link to approve there), so changed that. But also the rest of the code only worked for ideation, so simplified it as the FE always routes to the correct place.